src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja

```diff
@@ -84,25 +82,32 @@
 			return (unit == 'sat' ? amount / 100000000 : amount) > parseFloat('{{ wallet.full_available_balance }}');
 		}
 
+		function isInvalidValue(unit){
+			return (unit == 'sat' ? amount / 100000000 : amount) < 1e-8;
+		}
+
 		// Form validation
 		function validateAmount() {
 			let amountInput = document.getElementById('amount');
 			amountInput.max = getSpendableAmount(unit);
 
 			if (isAboveWalletBalance(unit)) {
```

Comment:
This gives a wrong unit error even if I just enter the address before entering the amount.
I think adding this check as the first condition before this line will make it better:
```
if (!amount) {
    return false;
}
```

Reply:
done :)